### DISCUSSION TITLE
[Game Support] StarCraft II: Legacy of the Void

### DISCUSSION THREAD

**Discussion by paranormalraccoon** [etournity/etournity#76] [Game Support]
### 2. Genre

RTS

### 3. How are tournaments for this game currently organized?

What is used to organize? (Example -> start.gg, spreadsheets, Pen and Paper)

- spreadsheets
- Discord
- Challonge

Which game modes are being played?

-

What are usual tournament formats? (Example -> Singe/Double Elimination, Swiss Style, ...)

- 1v1 Single Elimination
- 1v1 Double Elimination
- 

### 4. Platforms

PC

### 5. Additional Information

- organizers need to know which faction (Zerg, Protoss, Terran) each player wants to play for the whole tournament

**Comment by FLuplow** [comment]
Here is a small write-up of what needs to happen until people can make tournaments for this game (not final):
- #### General
  - we need to add Starcraft as a `Game` to our db
  - we should check all occurrences of Brawlhalla (to see if they are hard-coded)
  - we should check occurrences of game names for overflow (the full name of the game is way longer than Brawlhalla)
- #### Creation
  - the `Platform' step -> SC2 is PC only (this could be removed altogether)
  - the `Region` step -> currently specific to Brawlhalla -> should be Game specific
- #### Registration
  - race input
    - as an additional field in `GameUser` for game-specific info (e.g. "extra" as JSON -> {"race": "Zerg" | "Protoss" | "Terran"})
    - as a string addition to the `inGameName` (additional field in registration modal, but concatenated to the ign field)
- #### Host Hub
  - host and moderators should be able to change players `inGameName` to add their played race
- #### Matchlobby
  - there is no lobby code, players invite each other to a party and create a game (might need to replace the screen with an explanation)
  - the instructions / tutorial at the bottom is Brawlhalla specific (could just be removed altogether)

**Comment by paranormalraccoon** [comment]
Regarding Matchlobby:
- we could ask for the battle tag during tournament registration
- battletag is then shown in Matchlobby to Lobby Creator, who has an additional step of sending a friend request to opponent
- after friend request is accepted, opponent can be invited to in-game lobby
- battletag format: "name#1234"
  - can be found by clicking on "person" icon on bottom right 

![grafik](https://github.com/etournity/etournity/assets/61653882/f35a1c87-0927-46f1-9920-c6b3cdc4150d)
![grafik](https://github.com/etournity/etournity/assets/61653882/42f3045b-882c-4d08-a9e9-f9e3238ef124)

**Comment by paranormalraccoon** [comment]
We could remove the option for platform-choosing alltogether until we actually need it for a game. Also needs to be removed on Tournament List Card + Info Page, etc.

**Comment by paranormalraccoon** [comment]
For the first version, we should remove the "region" option, to make implementation quicker. Organizers could still use the Infobox for clarification of regions.
